<map id="openage::error::Error::store_cause" name="openage::error::Error::store_cause">
<area shape="rect" id="node2" href="$d0/df4/classopenage_1_1error_1_1_error.xhtml#a755a7891dfe08412bc2312f29b0e618d" title="openage::error::Error\l::Error" alt="" coords="211,5,353,47"/>
<area shape="rect" id="node3" href="$d9/dde/namespaceopenage_1_1pyinterface.xhtml#a89529648faa2ef530fefcc90f1c1ab70" title="Analyzes the current Python exception, and creates and throws an analogous C++ PyException object..." alt="" coords="195,71,369,112"/>
<area shape="rect" id="node4" href="$d0/d44/classopenage_1_1pyinterface_1_1_func.xhtml#acea6cb67bc0e8ea78d513f6db4e2e477" title="Specialization for bind() with void return types. " alt="" coords="417,63,620,119"/>
</map>
